source/user_defined_output/user_defined_output_plugin_creator.cc

```diff
@@ -0,0 +1,30 @@
+#include "source/user_defined_output/user_defined_output_plugin_creator.h"
+
+#include "envoy/config/typed_config.h"
+
+#include "external/envoy/source/common/common/statusor.h"
+#include "external/envoy/source/common/config/utility.h"
+
+namespace Nighthawk {
+
+using envoy::config::core::v3::TypedExtensionConfig;
+
+std::vector<UserDefinedOutputPluginPtr> createUserDefinedOutputPlugins(
+    std::vector<std::pair<TypedExtensionConfig, UserDefinedOutputPluginFactory*>>&
+        factory_config_pairs,
+    int worker_number) {
+  std::vector<UserDefinedOutputPluginPtr> plugins;
+
+  for (std::pair<TypedExtensionConfig, UserDefinedOutputPluginFactory*>& pair :
```

Comment:
using auto here might make this less verbose.

Reply:
Sure, given that the type is very clearly expressed above, I think that's an easy improvement